apps/console/src/features/authentication/store/actions/authenticate.ts

```diff
@@ -255,6 +256,13 @@ export const initializeAuthentication = () => (dispatch) => {
         // Update the context with new config once the basename is changed.
         ContextUtils.setRuntimeConfig(Config.getDeploymentConfig());
 
+        // Update post_logout_redirect_uri of logout_url with tenant qualified url
+        if (sessionStorage.getItem(LOGOUT_URL)) {
+            let logoutUrl = sessionStorage.getItem(LOGOUT_URL);
+            logoutUrl = logoutUrl.replace(window["AppUtils"].getAppBase() , window["AppUtils"].getAppBaseWithTenant());
```

Comment:
Possible max-length issue here. Please check

Reply:
The length is 119 character. So it should be okay.